DependencyInjection/SonataDoctrinePHPCRAdminExtension.php

```diff
@@ -89,6 +89,20 @@ public function getNamespace()
      */
     private function loadTreeTypes($config, ContainerBuilder $container)
     {
+        @trigger_error(
+            'The "sonata_doctrine_phpcr_admin.document_tree_options" configuration is deprecated since 1.2.8, to be replaced in 2.0.',
+            E_USER_DEPRECATED
+        );
+
+        if (isset($config['document_tree_defaults'])) {
+            @trigger_error(
+                'The "sonata_doctrine_phpcr_admin.document_tree_defaults" configuration is deprecated
+                since 1.2.8, to be moved in 2.0.
+                Use "sonata_doctrine_phpcr_admin.document_tree.routing_defaults" then',
```

Comment:
You should introduce `sonata_doctrine_phpcr_admin.document_tree.routing_defaults` in this PR and remove " then". Same goes for the other deprecations